### DISCUSSION TITLE
how to pass docker run args to docker-slim

### DISCUSSION THREAD

**Discussion by cr-ItaySadovnik** [slimtoolkit/slim#303] [Q&A]
Hi all,
I'm a docker-slim newbie,
how can i pass docker run command like --privileged or --cap-add to docker-slim
10x

**Comment by kcq** [comment]
Technically the `--privileged` is already used (when the temporary container is created).

There's no direct `--cap-add` flag, but you can use the `--cro-host-config-file` flag where you can point to a host config file where you can specify a list of `CapAdd` values ( https://pkg.go.dev/github.com/fsouza/go-dockerclient#HostConfig ).
